unity/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -5,9 +5,12 @@ EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
   m_Scenes:
-  - enabled: 1
+  - enabled: 0
     path: Assets/Scenes/DeviceData.unity
     guid: a85ce1c6384d24c2a8e016dbf92ebe5c
+  - enabled: 1
+    path: Assets/Scenes/XiheDemo.unity
+    guid: 16bf2390a4afb41c48a28fbea06c2cdd
```

Comment:
I think we should keep the default scene as `DeviceData.unity` on the main branch. Let's revert these changes back.

Reply:
Sounds good, I'll make sure of this in later changes